impls/gsnforwarder/contracts/@openzeppelin/contracts/interfaces/IERC165.sol

```diff
@@ -0,0 +1,6 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts v4.4.1 (interfaces/IERC165.sol)
```

Comment:
I was originally confused, thinking that they combined multiple OpenZeppelin versions, but it turns out that OpenZeppelin only updates the version when they change the contract. We still don't know what version they used, but it must be >= `v4.7.3`.